components/subscribe-to-stanford/Component.jsx

```diff
@@ -53,27 +51,28 @@ export default function SubscribeToStanfordReport({ contentConfiguration }) {
       onSubmit={submitSubscription}
       method="POST"
     >
-      <h2 className="su-m-0 su-font-bold su-text-18 su-leading-[2.16rem] dark:su-text-white su-font-sans">
+      <h2 className="su-m-0 su-font-bold su-text-18 su-leading-display dark:su-text-white su-font-sans">
         {title}
       </h2>
 
-      <p className="su-m-0 su-font-normal su-text-16 su-leading-[1.25] dark:su-text-white">
+      <p className="su-m-0 su-text-16 su-leading-[125%] dark:su-text-white">
         {summary}
       </p>
 
       <input
         type="email"
-        name="subsciption-email"
+        name="subscription-email"
```

Comment:
NOTE: There is a typo here which I updated - please check that I didn't bring other things in case there are scripts that are using the wrong spelling elsewhere 👀  @broleomo